LED/LED.c

```diff
@@ -1,68 +1,60 @@
 // led.c
 #include "LED.h"
 
-void init_led(LED *led) {
-    gpio_init(led->pin);
-    gpio_set_dir(led->pin, GPIO_OUT);
-    gpio_put(led->pin, 0);
-}
-
-bool check_led_state(LED *led) {
-    if (gpio_get(led->pin)) {
-        return 1;
-    } else {
-        return 0;
-    }
-}
-
-void blink_led(LED *led, bool state) {
-    gpio_put(led->pin, state);
-}
-
-void led_toggle(LED led) {
-    gpio_put(led.pin, !check_led_state(&led));
-}
+//void init_led(LED *led) {
+//    gpio_init(led->pin);
+//    gpio_set_dir(led->pin, GPIO_OUT);
+//    gpio_put(led->pin, 0);
+//}
+//
+//bool check_led_state(LED *led) {
+//    if (gpio_get(led->pin)) {
+//        return 1;
+//    } else {
+//        return 0;
+//    }
+//}
+//
+//void blink_led(LED *led, bool state) {
+//    gpio_put(led->pin, state);
+//}
+//
+//void led_toggle(LED led) {
+//    gpio_put(led.pin, !check_led_state(&led));
+//}
 
 static uint16_t pwm_level = INIT_PWM_LEVEL;
```

Comment:
If you're reverting back to ```struct```, than ```pwm_level``` global variable is redundant:
save the level in the struct.

Reply:
> If you're reverting back to `struct`, then `pwm_level` global variable is redundant: save the level in the struct.

done!